src/app/c++/ZoneTransit.h

```diff
@@ -7,6 +7,7 @@
 
 #include <vector>
 #include <mutex>
+#include <boost/python/tuple.hpp>
```

Comment:
@maym2104  Est-ce qu'une approche comme [cela](http://stackoverflow.com/questions/16497889/how-to-expose-stdpair-to-python-using-boostpython) serait plus simple ? Cela éviterait un couplage fort entre l'implémentation et le fait que c'est exposé en python.